src/components/radioButton.tsx

```diff
@@ -46,14 +46,16 @@ function RadioButton({
     <label
```

Comment:
I am trying to remove this wrapping label but if I replace it by a div for example the radio input is no more clickable, only the label is. If someone has an idea why, please let me know 👍

Reply:
what happens if you remove the htmlFor as well?

Reply:
Same, the input is still not clickable

Reply:
I quickly checked and for me it's all working fine. The issue that I can see that in the radioButton.stories.tsx there is no `name` passed as a prop. As this name is going to be the ID of the radio button and the `htmlFor` of the label, it is mandatory. If I pass name in stories, it works as expected.

Reply:
As you suggested the issue was due to CSS. We don't actually use the native input but a custom one. I gave the native input a width and a height and I set a z-index for it to be triggered on click.